Packages/com.unity.inputsystem/InputSystem/Editor/UITKAssetEditor/Views/ActionMapsView.cs

```diff
@@ -101,11 +108,6 @@ private void ChangeActionMapName(int index, string newName)
             Dispatch(Commands.ChangeActionMapName(index, newName));
         }
 
-        private void SelectActionMap()
-        {
-            Dispatch(Commands.SelectActionMap((string)m_ListView.selectedItem));
-        }
```

Comment:
A Coding-guideline question: I think outsourcing of code snippets can increase the readability. But that is a style decision.

Reply:
I think that's a valid comment but I also noticed this was the only place I could see where that function wasn't inlined with a lambda (comparing other view classes). Since this function was only called in a single place I considered it to be more readable when put in context with the triggering event callback (instead of having to lookup who is actually calling SelectActionMap) and also avoids anyone to call the function (removed from scope). I can move it back into a function as `void SelectActionMap(IEnumerable<int>) ` if you'd like?

Reply:
Good point, I didn't consider the aspect of the extended scope, I think it is good to avoid unwanted calls. Also if it is consistent to the other view classes you are right to keep it that way. I might have had a wrong perception of it. _Personally_ I like it more to split things up and give meaningful method names to make it readable. Now I am convinced to leave it as it is due to the reasons you mentioned.